packages/kotti-ui/source/next/kotti-field-radio-group/KtFieldRadioGroup.vue

```diff
@@ -78,7 +79,6 @@ export default defineComponent({
 		return {
 			inputProps: computed(() => ({
 				...field.inputProps,
-				class: 'kt-field-radio-group__wrapper__input',
```

Comment:
@tomquinonero this line makes me think that, combined with the fact that this doesn't happen on current master, so it can only come from this branch.

Reply:
@FlorianWendelborn :cry: I get it now. Is there any benefit using that over just putting the class in the template? It might make sense to you but it sure is not to me and we lost quite some time over that... Thoughts?

Reply:
Templates don't support type checking, we have this on every field, and BEM classes work well for everything else so why'd we suddenly use a different selector type?